examples/experimental/dagster-airlift/dagster_airlift/core/serialization/compute.py

```diff
@@ -25,8 +33,12 @@ class AirliftMetadataMappingInfo:
     asset_specs: List[AssetSpec]
 
     @cached_property
-    def mapped_asset_specs(self) -> List[AssetSpec]:
-        return [spec for spec in self.asset_specs if is_mapped_asset_spec(spec)]
+    def mapped_to_task_asset_specs(self) -> List[AssetSpec]:
+        return [spec for spec in self.asset_specs if is_task_mapped_asset_spec(spec)]
+
+    @cached_property
+    def mapped_to_dag_asset_specs(self) -> List[AssetSpec]:
```

Comment:
Feels like there's room for consolidation here: you could imagine consolidating this property into _all_ the mapped specs, dag or task.